src/compile/selection/transforms/project.ts

```diff
@@ -63,42 +68,46 @@ const project: TransformCompiler = {
       return {[range]: sg};
     };
 
+    const type = selCmpt.type;
+    const init = selDef.value;
```

Comment:
```suggestion
    const {type, init} = selCmpt;
```

Reply:
Wait no, this change is incorrect.

Reply:
Ups, yes. Should be:

```
const {type} = selCmpt;
const {value: init} = selDef;
```

Reply:
Not sure I understand why we prefer that? Your suggestion is harder for me to read/understand.

Reply:
Your original code is fine. I don't have strong preferences either way.